### PR TITLE
scripts: builds Docker image

### LINKED ISSUE #50: docker image
docker image for pulling zetcd

### PR DESCRIPTION
**Pull request by kruppel**
Adds scripts to build minimal Docker images for release.

Fixes #50.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.4
+
+COPY _output/bin/zetcd /usr/local/bin/zetcd
```
Comment (xiang90) [inline Dockerfile]: prefer ADD over COPY

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.4
```
Comment (F21) [inline Dockerfile]: How about using alpine 3.6, which is the latest release?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+export GO15VENDOREXPERIMENT=1
+
+LD_FLAGS="-w -X $(REPO_PATH)/version.Version=$(VERSION)"
+
+release-binary:
+	@go build -o $(GOPATH)/bin/zetcd -v -ldflags $(LD_FLAGS) $(REPO_PATH)/cmd/zetcd
+
+_output/bin/zetcd:
+	@./scripts/docker-build
+	@sudo chown $(user):$(group) _output/bin/zetcd
```
Comment (heyitsanthony) [inline Makefile]: I tried running `docker-build` and it seems the `docker cp` will copy into a file owned by the user invoking it, so is this chown needed?

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+export GOBIN=$(PWD)/bin
+export PATH=$(GOBIN):$(shell printenv PATH)
+export GO15VENDOREXPERIMENT=1
+
+LD_FLAGS="-w -X $(REPO_PATH)/version.Version=$(VERSION)"
+
+release-binary:
+	@go build -o $(GOPATH)/bin/zetcd -v -ldflags $(LD_FLAGS) $(REPO_PATH)/cmd/zetcd
+
+_output/bin/zetcd:
```
Comment (heyitsanthony) [inline Makefile]: `bin/zetcd-docker:`?

**Comment by heyitsanthony** [comment]
@kruppel ping?

**Comment by F21** [comment]
friendly ping @kruppel

**Comment by kruppel** [comment]
@heyitsanthony @F21 Sorry for the delay. PR updated with changes (hopefully) addressed. Let me know if any further updates are required. Thanks!

**Pull request merged**